src/style.css

```diff
@@ -0,0 +1,59 @@
+body {
+  margin: 0 auto;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  position: relative;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;
+  position: relative;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  position: relative;
+}
+
+.stars_star:not(:first-child) {
+  margin-left: 4px;
```

Comment:
it's the star that should have a background image and not the `:after` pseudo-element